src/services/env-utils.ts

```diff
@@ -0,0 +1,9 @@
+export const getEnvVar = (envVar: string, defaultValue: string) => {
+  const value = process.env[envVar];
```

Comment:
I was wondering (and need your thoughts on that one too please) if we wanted to have some config object we want to share across the service at build time. It seems that we call/use`process.env` on multiple files (secret.ts. configuration.ts, etc..) so i was wondering if we returned s config object instead (that shared) of calling `process.env` and log once the app is running would be easier to maintain and troubleshoot. Up to you but wanted to share that thought with you

Reply:
Well, the only usage of `process.env` is on this line (in the entire project). `secrets.ts` was deleted a while ago in favour of configuration-as-code. In addition, we already have a configuration object shared across the service at build time -  repeated calls to `configuration()` (from `configuration.ts`) are idempotent.

As to logging the config - we think this'd be unnecessary, as the configuration is already readily available on GitHub (same file, `configuration.ts`). When debugging a branch, you know that the configuration object is exactly identical to the one on that branch in GitHub. :) we think the current implementation is a good compromise